flycheck.el

```diff
@@ -888,6 +888,18 @@ Set this variable to nil to disable the mode line completely."
   :risky t
   :package-version '(flycheck . "0.20"))
 
+(defcustom flycheck-mode-line-lighter-base "FlyC"
```

Comment:
What about `flycheck-mode-line-prefix`?  It's shorter, and that's what it is, isn't it?